KioskApplication/src/main/java/controller/map/MapController.java

```diff
@@ -50,19 +66,23 @@
     @FXML private JFXCheckBox showNodesBox;
     @FXML private JFXCheckBox showEdgesBox;
 
-    private Path currentPath;
+    @FXML private ObservableList<javafx.scene.Node> visibleWaypoints;
```

Comment:
I don't think this variable is ever used? If we have the PathWaypointView then shouldn't that handle everything about waypoints?